### ISSUE TITLE
Expand and Fix Conditional formaating on the Accessibility Tracking Worksheet

### ISSUE THREAD

**Issue by CM805** [ONRR/onrr.gov-site#885]
I realized we needed to add conditional formatting for "Sent" and "Not Accessible" to be red.  I've worked on it off and on this morning.  Came back to it and it's not working properly.  @Baymatt can you please take a look and try to get it corrected?

**Comment by CM805** [comment]
@Baymatt thought it might also be good to add data validation to column with the status

**Comment by mcharg** [comment]
@CM805 I downloaded the Accessibility tracking sheet and sent it to Matt.  When he completes the edits, we'll need to check to see if changes have been made to the spreadsheet in OneDrive before uploading the version with edits.

@cthomasONRR @daniel-coleman-1 @Maroyafaied FYI - If you make edits to the Accessibility Tracking sheet before this is closed, let @CM805 know.

**Comment by mcharg** [comment]
@CM805 This looks like it was updated in the spreadsheet.  Can we close this issue?

**Comment by cthomasONRR** [comment]
@mcharg I believe this issue can be closed. 

We are now using Matt's new excel sheet (https://app.zenhub.com/workspaces/data-display-5464fa79c3cc8d7f4c384eaf/issues/onrr/onrr.gov-site/841)

And we made a new file in the 508 One Drive to house older versions of the excel tracker (https://doimspp-my.sharepoint.com/:f:/r/personal/mchargs_mms_gov/Documents/onrr.gov/508/Archive%20for%20AccessibilityTracking%20Excel?csf=1&web=1&e=rS4sOr)

**Comment by mcharg** [comment]
Thanks for the update @cthomasONRR!
